AssetManagementTestProject/ActualData/AssetActualData.cs

```diff
@@ -0,0 +1,65 @@
+﻿using CoreFramework.DriverCore;
+using OpenQA.Selenium;
+using AssetManagementTestProject.DAO;
+
+namespace AssetManagementTestProject.ActualData;
+/// <summary>
+/// Iterate through each cell of the View Asset Grid
+/// Add texts from each cell to an attribute of the AssetDAO
+/// Return an AssetDAO object or the JSON-converted version
+/// Similar for other ActualData classes
+/// </summary>
+public class AssetActualData : WebDriverAction
+{
+
+    public AssetActualData(IWebDriver? driver) : base(driver)
+    {
+    }
```

Comment:
Why do we need the driver here? I don't see the reference?
Why do we need to inherit the WebDriverAction?

Reply:
I have just figured out how to call for WebDriverAction(stringbaseUrl = "") yesterday so I have yet to have time to change all Page Objs. I am fixing this ASAP